released/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.1.0.0/opam

```diff
@@ -0,0 +1,66 @@
+opam-version: "2.0"
+maintainer: "Reynald Affeldt <reynald.affeldt@aist.go.jp>"
+
+homepage: "https://github.com/math-comp/analysis"
+dev-repo: "git+https://github.com/math-comp/analysis.git"
+bug-reports: "https://github.com/math-comp/analysis/issues"
+license: "CECILL-C"
+
+synopsis: "An analysis library for mathematical components"
+description: """
+This repository contains an experimental library for real analysis for
+the Coq proof-assistant and using the Mathematical Components library."""
+
+build: [make "-C" "theories" "-j%{jobs}%"]
+install: [make "-C" "theories" "install"]
+depends: [
+  "coq-mathcomp-classical" { = version}
+  "coq-mathcomp-solvable" { (>= "2.0.0") | (= "dev") }
```

Comment:
```suggestion
  "coq-mathcomp-solvable" {>= "2.0.0"}
```
The current formulation is redundant, since `>= "X.Y.Z"` (where `X`, `Y`, `Z` are numbers) will always include `"dev"`.